hack/create.sh

```diff
@@ -75,7 +74,6 @@ properties:
         tenantId: $AZURE_TENANT_ID
   routerProfiles:
   - name: default
-    publicSubdomain: $RESOURCEGROUP.$DNS_DOMAIN
```

Comment:
@jim-minter this is "Defaulting code". As I understood we leaving this for Post PP?

Reply:
@charlesakalugwu if you remove this line, which you probably should, you'll need to add logic into enrich() to populate it.